test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -347,6 +347,14 @@ function compute-kube-apiserver-params {
 	fi
 	params+=" --tls-cert-file=/etc/srv/kubernetes/server.cert"
 	params+=" --tls-private-key-file=/etc/srv/kubernetes/server.key"
+	params+=" --requestheader-client-ca-file=/home/kubernetes/k8s_auth_data/aggr_ca.crt"
```

Comment:
You need to set this path to `/etc/srv/kubernetes/aggr_ca.crt`. Likewise for others.
Because we copy files from `/home/kubernetes/k8s_auth_data` to that path somewhere before.